src/routes/recipient/handlers.js

```diff
@@ -161,3 +163,41 @@ export async function getRecipientLeadership(req, res) {
     await handleErrors(req, res, error, logContext);
   }
 }
+
+export async function getMergeGoalPermissions(req, res) {
+  try {
```

Comment:
Should we have this in goals route?

Reply:
I thought it belonged here since it gets called for a recipient. Maybe you could suggest a better name.